src/node/utils/ExportHtml.js

```diff
@@ -72,28 +70,27 @@ async function getHTMLFromAtext(pad, atext, authorColors) {
   const anumMap = {};
   let css = '';
 
-  const stripDotFromAuthorID = function (id) {
-    return id.replace(/\./g, '_');
-  };
+  const stripDotFromAuthorID = (id) => id.replace(/\./g, '_');
 
   if (authorColors) {
     css += '<style>\n';
 
     for (const a in apool.numToAttrib) {
       const attr = apool.numToAttrib[a];
+      let propName;
+      let newLength;
```

Comment:
Nit: The scope of these vars shouldn't be this high. Use `const propName` and `const newLength` in each `if` case.